modules/ROOT/pages/tutorial-manage-an-api.adoc

```diff
@@ -8,42 +8,55 @@ In this procedure, you configure and apply a rate limiting SLA-based policy to a
 * A tier that limits requests to three per minute, with no approval required
 * A tier that limits requests to five per minute, with API Versions Owner approval required
 
-When an app attempts to consume the protected API resource, the rate limiting SLA-based policy is enforced. Because this policy is client ID-based, app requests must include the expected client ID. The client secret is optional.
+When an app attempts to consume the protected API resource, the rate limiting SLA-based policy is enforced. Because this policy is client ID-based, app requests must include the expected client credentials. The procedure demonstrates including client credentials as headers, which is more secure than including the credentials as query parameters.
```

Comment:
```suggestion
When an app attempts to consume the protected API resource, the Rate Limiting SLA-based policy is enforced. Because this policy is client ID-based, app requests must include the expected client credentials. The procedure demonstrates including client credentials as headers, which is more secure than including the credentials as query parameters.
```

Reply:
Good catch. Fixed!